src/ui/screens/settings/ViewSubscriptionManagement/SubscriptionManagementItem.qml

```diff
@@ -157,71 +157,48 @@ ColumnLayout {
     }
 
     RowLayout {
-        id: relayUpsell
-        objectName: _objectName + "-relayUpsell-layout"
+        id: upgradeToAnnualSub
+        objectName: _objectName + "-upgradeToAnnualSub-layout"
 
         visible: false
         Layout.fillWidth: true
         Layout.topMargin: MZTheme.theme.listSpacing
         Layout.bottomMargin: MZTheme.theme.windowMargin
 
-        Component.onCompleted: {
-            if (visible) {
-                Glean.sample.bundleUpsellViewed.record();
-            }
-        }
-
-        ColumnLayout {
+        MZBoldInterLabel {
+            color: MZTheme.colors.purple70
+            horizontalAlignment: Text.AlignLeft
+            font.pixelSize: MZTheme.theme.fontSizeSmall
+            text: MZI18n.SubscriptionManagementUpgradeToAnnual // "Save 50%..."
+            wrapMode: Text.WordWrap
+            lineHeight: 13
+            Layout.alignment: Qt.AlignLeft
             Layout.fillWidth: true
-
-            MZInterLabel {
-                color: MZTheme.theme.fontColorDark
-                horizontalAlignment: Text.AlignLeft
-                font.pixelSize: MZTheme.theme.fontSizeSmall
-                text: MZI18n.SubscriptionManagementAddFirefoxRelay // "Add Firefox Relay"
-                wrapMode: Text.WordWrap
-                lineHeight: 13
-                Layout.alignment: Qt.AlignLeft
-                Layout.fillWidth: true
-                elide: Text.ElideRight
-            }
-
-            MZLinkButton {
-                objectName: _objectName + "-relayUpsell-learnMoreLink"
-                linkColor: MZTheme.theme.blueButton
-                fontSize: MZTheme.theme.fontSizeSmall
-                labelText: MZI18n.SplittunnelInfoLinkText // "Learn more"
-                Layout.alignment: Qt.AlignLeft
-                padding: 4
-                Layout.leftMargin: -4
-
-                onClicked: {
-                    Glean.sample.bundleUpsellLinkClicked.record();
-                    MZUrlOpener.openUrlLabel("relayPremium");
-                }
-            }
+            elide: Text.ElideRight
```

Comment:
No need for the `elide` property if we are expecting this label to wrap with no `maximumLineCount`